sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -132,6 +132,17 @@ struct _pi_device : _ur_device_handle_t {
   using _ur_device_handle_t::_ur_device_handle_t;
 };
 
+// Structure describing the command queue descriptor when command lists are
+// created. This is used to ensure that the appropriate command list type is
+// reused from the context's cache.
+struct pi_command_list_and_desc_t {
+  ze_command_list_handle_t CmdList;
```

Comment:
why are you adding this? It is already the key in the `Queue->CommandListMap` right?

Reply:
oh, I see. you need this for use from context's cache where there is no `Queue->CommandListMap`